source/_components/media_player.samsungtv.markdown

```diff
@@ -34,6 +34,7 @@ Configuration variables:
 - **name** (*Optional*): The name you would like to give to the Samsung Smart TV.
 - **timeout** (*Optional*): The time-out in seconds for the communication with the TV. Defaults to 0 (no timeout).
 - **mac** (*Optional*): The MAC address of the Samsung Smart TV, eg. `00:11:22:33:44:55:66`. Required for power on support via wake on lan.
+- **get_status_via_rest** (*Optional*): Use samsung REST end point to determine the status of the TV (host:8001/api/v2/), eg. `True` or `False`. Required for latest 2017 model that are turned ON randomly (sending a key to check status turned the TV ON)
```

Comment:
**Spelling & Grammar**:
- `samsung` -> `Samsung`
- `end point` -> `endpoint`
- `eg.` -> `e.g.`
-

Reply:
@frenck sure, I made the requested change, thanks for the review :smile: